src/components/LocationAutocomplete/LocationAutocomplete.tsx

```diff
@@ -199,23 +199,22 @@ const LocationAutocomplete: React.FC<Props> = ({
     };
 
     return (
-        <AutosuggestDeprecated
-            getSuggestions={suggestionsList}
+        <Autosuggest
+            suggestions={suggestionsList}
             suggestionToString={suggestionToString}
             isLoading={isLoading}
-            isFocused={isFocused}
-            defaultInputValue={defaultInputValue}
+            defaultValue={defaultInputValue}
             inputPlaceholder={inputPlaceholder}
             showClearButton={singleLocation}
             clearTitle={clearTooltipLabel}
             noSuggestionsPlaceholder={noSuggestionsPlaceholder}
             listRenderer={renderListPoweredByGoogle}
-            saveSelectedValueToInput={singleLocation}
```

Comment:
Autosuggest doesn't contain saveSelectedValueToInput but it contains selectedSuggestions, I'll apply it but I'm not sure about it

Reply:
did you check the expected behaviour based on the tests and storybook? The best way if you understand the goal/requirements. Just looking at the code, might not be enough